### ISSUE TITLE
Using react-motion is a bad idea!

### ISSUE THREAD

**Issue by eldonwilliams** [eldonwilliams/what-is-a-computer#1]
The goal of what-is-a-computer was to create a simple website for a school project; however, one of the methods I used to implement animation is highly inefficient for mobile. While react-motion is an awesome introduction to spring-based animation and moving things away from CSS, it is not the best. For you who don't know react-motion runs the calculations for every frame using `requestAnimationFrame()` and the CPU, both of which are highly inefficient. They do work well on PC, however, I still do notice frame drops sometimes. CSS is efficient because all the calculations for the Tween states are done on the GPU in a multi-threaded fashion. I want to take advantage of this by changing the animation library to framer-motion.

## What does this mean for what-is-a-computer?

Two main things
- react-motion will be removed from the project and framer-motion will be added
- The component responsible for handling the slides will be changed to take advantage of framer-motion

### Extra notes:
This will also come with a few new features! I want to add swipe-to-change and arrows. Framer-motion will make that first one much easier because of the built-in dragging features.

TLDR;

We're reworking the slide change animation to be more efficient for low-end devices by taking advantage of the GPU via framer-motion. Changing slides will also support the framer-motions drag function.
